tendermint/starport
# fix(scaffolding): correct modified/created file logs

closes #1340

### What does this MR does?

Fix scaffold commands outputs extra file modification output.

### Solution

- Check if the `testutils` folder exists before box them

### How to test?

Run: `starport s list user`

Check if these below logs are not printed:

```shell
modify testutil/network/network.go
modify testutil/simapp/simapp.go
```

remove the `testutil`, runs again, and check if the logs above exist
